### PR TITLE
[Issue-3463] Extension - Add validate for Solochain when receive, transfer with Generic ledger account

### PR DESCRIPTION
**Pull request by S2kael**
### Related issue(s)

- #3463

### Is your feature request related to a problem(s)? Please describe.

- Extension - Add validate for Solochain when receive, transfer with Generic ledger account

### Describe the solution you'd like

- Extension - Add validate for Solochain when receive, transfer with Generic ledger account

### Describe alternatives you've considered

- No

### Additional context

- No

### CODE REVIEW

**Comment by saltict** [comment]
🚀 Deployed on https://pr-3464--sw-web-runner.netlify.app

File: packages/extension-koni-ui/src/constants/ledger.ts [hunk 1]
```diff
@@ -507,6 +507,8 @@ export const StandardDerivationPathGens: string[] = Object.values(PredefinedLedg
 
 export const NotNeedMigrationGens: string[] = [...PolkadotDerivationPathGens, ...StandardDerivationPathGens];
 
+export const AllowLedgerGenericNetworks: string[] = ['polkadot'];
```
Comment (lw-cdm) [inline packages/extension-koni-ui/src/constants/ledger.ts:510]: Please check this variable usage

File: packages/extension-koni-ui/src/components/TokenItem/TokenSelectionItem.tsx [hunk 2]
```diff
@@ -40,8 +42,22 @@ const Component = (props: Props) => {
     return reformatAddress(address || '', networkPrefix, isEvmChain);
   }, [address, chainInfo, symbol]);
 
+  const _onCLickCopyBtnAsync = useCallback((e: React.SyntheticEvent) => {
```
Comment (lw-cdm) [inline packages/extension-koni-ui/src/components/TokenItem/TokenSelectionItem.tsx:45]: onCLick => onClick

File: packages/extension-koni-ui/src/hooks/screen/home/useReceiveQR.ts [hunk 3]
```diff
@@ -186,7 +202,20 @@ export default function useReceiveQR (tokenGroupSlug?: string) {
       setTokenSelectorItems(_tokenSelectorItems);
 
       if (tokenGroupSlug) {
+        const firstToken = _tokenSelectorItems[0];
```
Comment (lw-cdm) [inline packages/extension-koni-ui/src/hooks/screen/home/useReceiveQR.ts:205]: Move this to into code block " if (_tokenSelectorItems.length === 1) "

**Pull request merged**